administration/src/bp-modules/user-settings/ChangePasswordForm.tsx

```diff
@@ -60,13 +61,27 @@ const ChangePasswordForm = (): ReactElement => {
           event.preventDefault()
           submit()
         }}>
-        <PasswordInput label={t('currentPassword')} value={currentPassword} setValue={setCurrentPassword} />
-        <PasswordInput label={t('newPassword')} value={newPassword} setValue={setNewPassword} />
-        <PasswordInput label={t('newPasswordConfirm')} value={repeatNewPassword} setValue={setRepeatNewPassword} />
-        {warnMessage === null ? null : <Callout intent='danger'>{warnMessage}</Callout>}
-        <div style={{ textAlign: 'right', padding: '10px 0' }}>
-          <Button text={t('changePassword')} intent='primary' type='submit' disabled={!valid} loading={loading} />
-        </div>
+        <Stack sx={{ gap: 2 }}>
+          <FormControl fullWidth>
+            <FormLabel>{t('currentPassword')}</FormLabel>
+            <PasswordInput value={currentPassword} setValue={setCurrentPassword} />
```

Comment:
when you start typing:
<img width="614" height="333" alt="Image" src="https://github.com/user-attachments/assets/2c1c837c-7d9b-4a47-bb7a-fb99a3b1d332" />

Reply:
https://github.com/digitalfabrik/entitlementcard/issues/2525